DOCKERHUB.md

```diff
@@ -0,0 +1,66 @@
+# Kroki
+
+Convert plain text diagrams to images !
+
+Kroki provides a unified API with support for BlockDiag (BlockDiag, SeqDiag, ActDiag, NwDiag, PacketDiag, RackDiag), BPMN, Bytefield, C4 (with PlantUML), Ditaa, Erd, Excalidraw, GraphViz, Mermaid, Nomnoml, Pikchr, PlantUML, SvgBob, UMLet, Vega, Vega-Lite, WaveDrom... and more to come!
+
+## Getting Started
+
+### Prerequisities
+
+In order to run this container you'll need docker installed.
+
+* [Windows](https://docs.docker.com/docker-for-windows/)
+* [OS X](https://docs.docker.com/docker-for-mac/)
+* [Linux](https://docs.docker.com/get-started/)
+
+### Usage
+
+See also the [installation](https://docs.kroki.io/kroki/setup/configuration/) docs in the [Kroki documentation](https://docs.kroki.io/)
+
+#### start a kroki instance
+
+`$ docker run --name some-kroki -d yuzutech/kroki`
+
+This image includes `EXPOSE 8000` (the kroki port), so standard container linking will make it automatically available to the linked containers. If you want to map port 8000 in the container to a port on your host, please use the `--publish` or `-p` flag:
+
+`$ docker run -p8000:8000 --name some-kroki -d yuzutech/kroki`
+
+#### connect with kroki-blockdiag
+
+We recommend using docker-composer to connect with [kroki-blockdiag](https://hub.docker.com/r/yuzutech/kroki-blockdiag) and/or [kroki-mermaid](https://hub.docker.com/r/yuzutech/kroki-mermaid)
+
+```
+version: "3"
```

Comment:
We should probably take this opportunity to add `bpmn` and `excalidraw` companion containers?

Reply:
👍 Added